src/components/Amalgamation/AmalgamationStatement.vue

```diff
@@ -77,6 +77,9 @@ export default class AmalgamationStatement extends Vue {
   /** Called when component is mounted. */
   mounted (): void {
     this.courtApproval = this.getAmalgamationCourtApproval
+    if (this.courtApproval !== null) {
```

Comment:
Does `if (this.courtApproval) this.amalgamationStatementValid(true)` work instead? I think that'll be cleaner!

Reply:
> Does `if (this.courtApproval) this.amalgamationStatementValid(true)` work instead? I think that'll be cleaner!

I don't think so, because the court approval value can still be false (select "without court approval"), which is still shoule be consider as valid :)

Reply:
How so? If the previous value was without court approval, shouldn't the validity be false?